### PR TITLE
Updating Swift 4.0 --> Swift 4.2

### PR DESCRIPTION
**Pull request by mduttondev**
* Took all migrator suggestions
* Built in a project locally and all seemed to work.

### CODE REVIEW

**Comment by mduttondev** [comment]
I opened this in hopes of getting this merged sooner rather than later. 

Also, PR [#409](https://github.com/vikmeup/SCLAlertView-Swift/pull/409) seems to be updating from `Swift 3.0` which seems unnecessary since there is a released version that supports `Swift 4.0`

File: SCLAlertView/SCLAlertView.swift [hunk 1]
```diff
@@ -977,7 +977,7 @@ open class SCLAlertView: UIViewController {
 
             let timeoutStr: String = showTimeout.prefix + String(Int(timeout.value)) + showTimeout.suffix
             let txt = String(btn.initialTitle) + " " + timeoutStr
-            btn.setTitle(txt, for: UIControlState())
+            btn.setTitle(txt, for: UIControl.State())
```
Comment (vikmeup) [inline SCLAlertView/SCLAlertView.swift]: Can you use `.normal` instead of `UIControl.State()`

Comment (vikmeup) [reply SCLAlertView/SCLAlertView.swift]: and in few more places

**Comment by mduttondev** [comment]
@vikmeup can you please promote this to cocoapods with the new version.

**Pull request merged**